lib/workload/stateless/stacks/filemanager/deploy/lib/filemanager.ts

```diff
@@ -53,36 +53,57 @@ export class Filemanager extends Stack {
   constructor(scope: Construct, id: string, props: FilemanagerProps) {
```

Comment:
@mmalenic I made tweaks on the  Filemanager Props here so that the lookup is now done within the stack. So this stack can independently run without depending on other stack. Perhaps I need a pair of eye if I did this correctly.